content/id/docs/concepts/storage/volumes.md

```diff
@@ -0,0 +1,1151 @@
+---
+title: Volumes
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+Berkas-berkas yang disimpan di _disk_ di dalam Container bersifat tidak permanen (akan terhapus seiring dengan dihapusnya Container/Pod), yang menimbulkan beberapa masalah untuk aplikasi biasa saat berjalan di dalam Container. Pertama, saat sebuah Container mengalami kegagalan, Kubelet akan memulai kembali Container tersebut, tetapi semua berkas di dalamnya akan hilang - Container berjalan dalam kondisi yang bersih. Kedua, saat menjalankan banyak Container bersamaan di dalam sebuah `Pod`, biasanya diperlukan untuk saling berbagi berkas-berkas di antara Container-container tersebut. Kedua masalah tersebut dipecahkan oleh abstraksi `Volume` pada Kubernetes.
+
+Pengetahuan akan [Pods](/docs/user-guide/pods) disarankan.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Latar Belakang
+
+Docker juga memiliki konsep _[volume]_(https://docs.docker.com/engine/admin/volumes/), walaupun konsepnya Docker agak lebih longgar dan kurang dikelola. Pada Docker, sebuah volume adalah sesederhana sebuah direktori pada _disk_ atau  di dalam Container lainnya. _Lifetime_ tidak dikelola dan hingga baru-baru ini hanya ada volume yang didukung _disk_ lokal. Docker sekarang menyediakan _driver_ untuk volume, namun fungsionalitasnya masih sangat terbatas (misalnya hingga Docker 1.7 hanya ada satu _driver_ volume yang diizinkan untuk setiap Container, dan tidak ada cara untuk menyampaikan parameter kepada volume).
+
+Sebaliknya, sebuah Volume Kubernetes memiliki _lifetime_ yang gamblang - sama dengan _lifetime_ Pod yang berisi Volume tersebut. Oleh karena itu, sebuah Volume bertahan lebih lama dari Container-container yang berjalan di dalam Pod tersebut, dan data di Volum tersebut juga dipertahankan melewati diulangnya Container. Tentu saja, saat sebuah Pod berakhir, Volume tersebut juga akan berakhir/terhapus. Dan mungkin lebih penting lagi, Kubernetes mendukung banyak jenis Volume, dan sebuah Pod dapat menggunakan sebanyak apapun Volume secara bersamaan.
+
+Pada intinya, sebuah volume hanyalah sebuah direktori, dan mungkin berisi data, yang dapat diakses oleh Container-container di dalam Pod. Bagaimana direktori tersebut dibuat, medium yang menyokongnya, dan isinya ditentukan oleh jenis volume yang digunakan.
+
+Untuk menggunakan sebuah volume, sebuah Pod memerinci volume-volume yang akan disediakan untuk Pod tersebut (kolom `.spec.volumes`) dan di mana volume-volume tersebut akan ditambatkan (di-_mount_) di dalam Container-container di Pod (kolom `.spec.containers.volumeMounts`).
```

Comment:
```suggestion
Untuk menggunakan sebuah volume, sebuah Pod memberikan perincian volume-volume yang akan disediakan untuk Pod tersebut (kolom `.spec.volumes`) dan di mana volume-volume tersebut akan ditambatkan (di-_mount_) di dalam kontainer-kontainer di Pod (kolom `.spec.containers.volumeMounts`).
```

Reply:
This is actually a standard word in KBBI. Or we can use "menguraikan" too. How about it?
 https://kbbi.kemdikbud.go.id/entri/memerinci